docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in **Admin** -> **Activity list**, you can consult what actions were taken by users recently.
+
+!!! note
+
+    - To see the Admin menu, you must have the "Setup / Administrate" [permission](permission_system.md).
+    - To see the Activity list, you must have the "Activity Log / Read" permission (this permission might be limitated to only see your own activity).
+
+By default, the following actions are logged:
```

Comment:
Honestly, I would get rid of all the links regarding actions. If you want to keep it, be more consistent - in some places links are missing, in some are wrong.
@juskora, @dabrt - what's your opinion?

Reply:
I guess instead of linking to individual actions, which are quite selectively described in our docs, we could link to higher-level places in doc:
```
[Content](.../content_management/content_items/)...
[Location](.../content_management/content_organization/manage_locations_urls/#content-locations)...
[Product](.../pim/products/)...
[Product variant](.../pim/work_with_product_variants/)...
[Site](.../website_organization/work_with_sites/)...
```

Reply:
Done in 1d184ca8ba70ccc031536c6c5732f2d5fdd6371b